concordium-consensus/src/Concordium/GlobalState/BlockState.hs

```diff
@@ -283,13 +283,25 @@ type family UpdatableContractState (v :: Wasm.WasmVersion) = ty | ty -> v where
   UpdatableContractState GSWasm.V0 = Wasm.ContractState
   UpdatableContractState GSWasm.V1 = StateV1.MutableState
 
+-- |An external representation of the persistent (i.e., frozen) contract state.
+-- This is used to pass this state through FFI for queries and should not be
+-- used for other purposes.
```

Comment:
Consider adding a brief explanation of _why_ it shouldn't be used for other purposes.

Reply:
I tried to do that.